qa/rpc-tests/mn_expiration.py

```diff
@@ -0,0 +1,577 @@
+            sell_ticket_txid, buy_ticket_txid
+        )
+
+        # fail if there is another trade ticket referring to that sell ticket
+        try:
+            self.register_nft_trade_ticket(
+                self.non_mn3, self.non_mn4, self.nonmn4_pastelid1, self.nonmn4_address1,
+                sell_ticket_txid, buy_ticket_txid
+            )
+        except JSONRPCException as e:
+            self.errorString = e.error['message']
+            print(self.errorString)
+        assert_equal("There is already exist trade ticket for the sell ticket with this txid [" +
+                     sell_ticket_txid + "]" in self.errorString, True)
+
+        if self.total_copies == 1:
+            try:
+                self.register_nft_sell_ticket(act_ticket_txid, 1)
+            except JSONRPCException as e:
+                self.errorString = e.error['message']
+                print(self.errorString)
+            assert_equal("The Art you are trying to sell - from registration ticket [" +
+                         act_ticket_txid + "] - is already sold - there are already [1] sold copies, "
+                         "but only [1] copies were available" in self.errorString, True)
+        elif self.total_copies == 2:
+            # fail if not enough copies to sell
+            try:
+                self.register_nft_sell_ticket(act_ticket_txid)
+            except JSONRPCException as e:
+                self.errorString = e.error['message']
+                print(self.errorString)
+            assert_equal("Invalid Sell ticket - copy number [3] cannot exceed the total number "
+                         "of available copies [2] or be 0" in self.errorString, True)
+
+            try:
+                self.register_nft_sell_ticket(act_ticket_txid, 1)
+            except JSONRPCException as e:
+                self.errorString = e.error['message']
+                print(self.errorString)
+            assert_equal("Cannot replace Sell ticket - it has been already sold. " +
+                         "txid - [" + sell_ticket_txid + "] copyNumber [1]" in self.errorString, True)
+
+            # fail as the replace copy can be created after 5 days
+            try:
+                self.register_nft_sell_ticket(act_ticket_txid, 2)
+            except JSONRPCException as e:
+                self.errorString = e.error['message']
+                print(self.errorString)
+            assert_equal("Can only replace Sell ticket after 5 days. txid - [" + sell_ticket3_txid + "] "
+                         "copyNumber [2]" in self.errorString, True)
+
+
+    # ===============================================================================================================
+    def initialize(self):
+        print("== Initialize nodes ==")
+
+        self.nodes[self.mining_node_num].generate(10)
+
+        # generate pastelIDs & send coins
+        self.artist_pastelid1 = self.nodes[self.non_mn3].pastelid("newkey", "passphrase")["pastelid"]
```

Comment:
I've created create_pastelid function in MasterNodeCommon class, so after we merge my LegRoast changes - this code can be updated

Reply:
also moved "passphrase" to class variable

Reply:
ok